modules/server_new/src/main/scala/observe/server/gmos/GmosControllerSim.scala

```diff
@@ -25,21 +27,18 @@ import observe.server.ObsProgress
 import observe.server.Progress
 import observe.server.ProgressUtil.countdown
 import observe.server.RemainingTime
-import observe.server.gmos.GmosController.Config.NSConfig
+import observe.server.gmos.GmosController.Config.NsConfig
 import observe.server.gmos.GmosController.GmosConfig
 import org.typelevel.log4cats.Logger
 
-import scala.concurrent.duration.Duration
-import scala.concurrent.duration.FiniteDuration
-
 /**
  * Keep track of the current execution state
  */
-final case class NSCurrent(
+final case class NsCurrent(
```

Comment:
I'd keep `NSCurrent`

Reply:
We had some `NS` and some `Ns`. To unify, I took the approach we usually see in other acronyms: `Http`, `Uri`, `Js`, etc.

Reply:
We use that convention for instruments too: `Gmos`, `Gnirs`, `Gsaoi`, etc.

Reply:
Ok, that seems fine